smartapp-core/src/main/java/com/smartthings/sdk/smartapp/core/extensions/contextstore/InstalledAppContextStore.java

```diff
@@ -0,0 +1,37 @@
+package com.smartthings.sdk.smartapp.core.extensions.contextstore;
+
+
+/**
+ * Implementations of this interface handle storing and updating
+ * InstalledAppContext instances, including keeping tokens up-to-date.
+ *
+ * IMPORTANT: Implementations of this class are responsible for keeping
+ * the tokens up-to-date. Refresh tokens expire after 30 days.
```

Comment:
Does the backing impl for exist yet?  Assuming it would need some sort of database / http client / etc?

Reply:
Yeah, I'm working on the first implementation (DynamoDB) right now. I wanted to get the API committed for review and to make the reviews smaller. So far, I have enough of it working to test that it's getting called at the correct times.